pallets/polkadex/src/tests.rs

```diff
@@ -38,7 +38,7 @@ fn setup_balances_test() {
     });
 }
 
-//Executes some pre-defined trades and checks if the order book state is as expected.
+// Executes some pre-defined trades and checks if the order book state is as expected.
 #[test]
 fn check_trading_engine() {
```

Comment:
Create a new function and modify values there.

Reply:
Alright.